### DISCUSSION TITLE
Is there something I have to do to enable the initial loading screen?

### DISCUSSION THREAD

**Discussion by greensourcecode** [NvChad/NvChad#1244] [Q&A] [answered]
I would like the ability to select new file, load file, etc
I would also like the terminal and filetree to open like in this screenshot:

![MORECHAD](https://user-images.githubusercontent.com/97241812/174521345-e437f52d-932a-4006-90a6-3a994b9de8be.png)

Sorry if this is a very novice question. If I'm not chad enough for NVIM yet I'll understand. . .

I do somewhat understand how NVIM works with Lua and a bit of Lua as well. I'm assuming I have to mess with the maybe the init.lua but I'm not sure how to accomplish these things and if it is any different than vanilla nvim.

**Comment by siduck** [comment]
@greensourcecode read the config docs to enable dashboard c:

**Reply by greensourcecode** [reply]
I actually did figure the dashboard. Thank you for letting me know what I was looking for. I'm going to figure out the terminal / file tree / explorer thing tomorrow morning and make sure I can get it set up correctly. Then I will share my findings and mark this answered 😎🤙

**Comment by greensourcecode** [comment, marked as answer]
What I was looking for what called the "dashboard"

So a better way of asking the question would have been "how do I enable the dashboard?"

or  "How do I make my nv-chad look more like the screenshots?"

I was able to do so by adding the following to my (custom) chadrc.lua file:

```
local M = {}

M.plugins = {
   user = {
      ["goolord/alpha-nvim"] = {
         disable = false,
      },
   },
}

return M
```

I found this information in the documentation [here](https://nvchad.github.io/config/Custom%20config).

I am learning to code on and older computer and I feel NVIM is my best option. I'm hoping nv-chad helps me get over the learning curve.

Thank you @siduck  for your help sir 😎 😎

Sorry for silly questions

**Reply by siduck** [reply]
@greensourcecode ask these stuffs on our community servers (discord or matrix) or telegram group. Check readme badge for link. I'll help there :)
